frontend/pubspec.yaml

```diff
@@ -26,12 +26,12 @@ dependencies:
   intl: ^0.16.1
   mapbox_gl: ^0.9.0
   provider: ^4.3.2+2
-  graphql_flutter: ^3.1.0
+  graphql_flutter: ^4.0.1
   url_launcher: ^5.7.10
   maps_launcher: ^1.2.2+2
-  flutter_svg: ^0.18.0
+  flutter_svg: 0.19.1
```

Comment:
Why not 
```suggestion
  flutter_svg: ^0.19.1
```

Reply:
Because I tested `0.19.2` and build failed due to incompatible SDK versions. But seems `flutter pub` is clever enough to still use `0.19.1`, so we can add the `^` 👍🏽